man/memkind.3

```diff
@@ -979,10 +979,13 @@ Controls logging mechanism in memkind. Setting
 to 1 enables printing messages like errors and general information about environment to
 .IR stderr .
 .TP
-.B MEMKIND_BACKGROUND_THREAD
-Enable background worker threads. Setting
-.B MEMKIND_BACKGROUND_THREAD
-to 1 enables background threads in jemalloc heap manager.
+.B MEMKIND_BACKGROUND_THREAD_LIMIT
+Enable background worker threads.
```

Comment:
Is it possible to disable background thread?

Reply:
Defining this variable is equal to enable background worker thread. Background thread is disabled in case you don't pass this environment variable.